pdns/dnsdistdist/docs/rules-actions.rst

```diff
@@ -610,6 +610,11 @@ These ``DNSRule``\ s be one of the following items:
 
    :param string qname: Qname to match
 
+.. function:: QNameSetRule(set)
+   Matches if the set contains qname.
```

Comment:
It might be nice to be explicit about the difference between `QNameSetRule` and `SuffixMatchNodeRule`, ie exact vs suffix match.

Reply:
Mentioning `QnameSetRule` in the documentation for `SuffixMatchNodeRule` might make sense as well.

Reply:
So I think we must make this difference very explicit, because it confused me too.

Reply:
Wrote more about the difference and add mentionings.